notifications.json

```diff
@@ -0,0 +1,504 @@
+                },
+                "dry_run": {
+                    "type": "boolean",
+                    "description": "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
+                },
+                "time_to_live": {
+                    "type": "integer",
+                    "description": "This parameter specifies how long (in seconds) the message should be kept in storage if the device is offline. The maximum time to live supported is 4 weeks, and the default value is 4 weeks.",
+                    "minimum": 0
+                },
+                "tags": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "content": {
+                    "$ref": "#/definitions/content"
+                }
+            },
+            "required": [
+                "to",
+                "content"
+            ]
+        },
+        "message-status": {
+            "title": "Message Status",
+            "type": "object",
+            "properties": {
+                "status": {
+                    "type": "string",
+                    "enum": [
+                        "PENDING",
+                        "SENDING",
+                        "SENT",
+                        "DELIVERED",
+                        "READ",
+                        "FAILED"
+                    ]
+                },
+                "error_message": {
+                    "type": "string"
+                },
+                "error_code": {
+                    "type": "string",
+                    "enum": [
+                        "PERMANENT_FAILURE",
+                        "TEMPORARY_FAILURE"
+                    ]
+                },
+                "service_id": {
+                    "minimum": 0,
+                    "type": "string"
+                },
+                "sent_at": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "delivered_at": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "read_at": {
+                    "type": "string",
+                    "format": "date-time"
+                }
+            }
+        },
+        "message-output": {
+            "title": "Message Output",
+            "allOf": [
+                {
+                    "type": "object",
+                    "properties": {
+                        "id": {
+                            "type": "string",
+                            "format": "uuid"
+                        },
+                        "created_at": {
+                            "type": "string",
+                            "format": "date-time"
+                        },
+                        "updated_at": {
```

Comment:
that's an internal status, it's not something that gets updated by an API client